client/my-sites/checkout/composite-checkout/components/additional-terms-of-service-in-cart.tsx

```diff
@@ -74,6 +74,40 @@ function getMessageForTermsOfServiceRecord(
 				args.subscription_expiry_date &&
 				args.subscription_auto_renew_date
 			) {
+				if (
+					args.is_renewal_price_prorated &&
+					( locale === 'en' ||
+						i18nCalypso.hasTranslation(
+							'The promotional period for your subscription lasts from %(startDate)s to %(endDate)s. On %(renewalDate)s we will charge your payment method (PAYPAL %(email)s) for %(renewalPrice)s. All subsequent renewals will be charged for the regular subscription price of %(regularPrice)s. You will receive at least one email notice %(numberOfDays)d days before being billed and can {{updatePaymentMethodLink}}update your payment method{{/updatePaymentMethodLink}} or {{manageSubscriptionlink}}manage your subscription {{/manageSubscriptionlink}} at any time.'
```

Comment:
> PAYPAL

I may be missing something obvious, but is this always a PayPal purchase?

Reply:
Yes, that is correct, we verify that the payment partner is PayPal on the backend.

Reply:
Oh! Sorry, I missed the `terms_for_bundled_trial_auto_renewal_paypal`.